dispatcher/dispatcher.go

```diff
@@ -408,8 +440,22 @@ func (d *IotxDispatcher) dispatchBlockSyncReq(ctx context.Context, chainID uint3
 	subscriber.ReportFullness(ctx, iotexrpc.MessageType_BLOCK_REQUEST, float32(l)/float32(c))
 }
 
+func (d *IotxDispatcher) handleBroadcastMsg(ctx context.Context, chainID uint32, peer string, message proto.Message) {
+	for _, sub := range d.msgSubscribers {
+		switch msg := message.(type) {
+		case *iotextypes.ConsensusMessage:
+			if err := sub.HandleMonitorMsg(msg); err != nil {
+				log.L().Debug("Failed to handle consensus message", zap.Error(err))
+			}
+		default:
+		}
+	}
+}
+
 // HandleBroadcast handles incoming broadcast message
 func (d *IotxDispatcher) HandleBroadcast(ctx context.Context, chainID uint32, peer string, message proto.Message) {
+	d.handleBroadcastMsg(ctx, chainID, peer, message)
```

Comment:
should return a flag to indicate if it's `iotextypes.ConsensusMessage`, if true can return, no need to continue